Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -96,7 +102,7 @@ def __init__(self, *args, **kwargs):
 		if not self.STAGE1B_XIP:
 			# For Stage1B, it can be compressed if STAGE1B_XIP is 0
 			# If so, 	STAGE1B_FD_BASE/STAGE1B_FD_SIZE need to be defined
-			self.STAGE1B_FD_SIZE    = 0x2C000
+			self.STAGE1B_FD_SIZE    = 0x30000
```

Comment:
If the size change is due to source level debug, then we should use the ENABLE_SOURCE_DEBUG to control it.  Otherwise, it will increase the size for all the build regardless of source level debug enable or not.

Reply:
Okay. I will do that.